models/junos/security_policies_global/junos_security_policies_global.yml

```diff
@@ -0,0 +1,211 @@
+---
+module: junos_security_policies_global
+version_added: 2.12
+short_description: Manage global security policy settings on Juniper JUNOS devices
+description: This module provides declarative management of global security policy settings on Juniper JUNOS devices
+author: Pranav Bhatt (pbhatt@redhat.com)
+requirements:
+  - ncclient (>=v0.6.4)
+notes:
+  - This module requires the netconf system service be enabled on the device being managed.
+  - (to be revisited)This module works with connection C(netconf). See L(the Junos OS Platform Options,https://docs.ansible.com/ansible/latest/network/user_guide/platform_junos.html).
+  - (to be revisited)Tested against JunOS v18.4R1
+options:
+  config:
+    description: A dictionary of security policies
+    type: list
+    elements: dict
+    suboptions:
+      default_policy:
+        description:
+          - Configure the default security policy that defines the actions the device takes on a packet that does not match any user-defined policy.
+        choices:
+          - deny-all
+          - permit-all
+      policy_rematch:
+        description:
+          - Enable the device to reevaluate an active session when its associated security policy is modified. The session remains open if it still matches the policy that allowed the session initially.
+        type: dict
+        suboptions:
+          extensive:
+            description:
+              - When a policy is modified or deleted, extensive option checks if any suitable policy permit to keep these sessions alive.
+            type: bool
+      policy_stats:
+        description:
+          - Configure systemwide policies statistics.
+        type: dict
+        suboptions:
+          system_wide:
+            description:
+              - Configure systemwide policies statistics.
+            type: bool
+      pre_id_default_policy:
+        description:
+          - During the initial policy lookup phase, which occurs prior to a dynamic application being identified, if there are multiple policies present in the potential policy list, the SRX Series device applies the default security policy until a more explicit match has occurred. Configures default policy actions that occur prior to dynamic application identification (AppID).
+        type: dict
+        suboptions:
```

Comment:
This param is takes input in the form of dict of dict, can't we make it a single dict as the parent param just seems to cater policy_action dict